src/main/webapp/app/admin/admin.route.ts

```diff
@@ -57,7 +58,14 @@ export const adminState: Routes = [
                 },
             },
             {
-                path: 'jhi-metrics',
+                path: 'docs',
```

Comment:
I can still access this route directly even if the openapi profile is not active.

Reply:
I guess this is ok for now. I’m not sure if we actually need this profile anyway